packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -167,9 +169,131 @@ test.serial('stakeAtom - smart wallet', async t => {
+  await wd.executeOffer({
+    id: 'request-deposit-success',
+    invitationSpec: {
+      source: 'continuing',
+      previousOffer: 'request-account',
+      invitationMakerName: 'Deposit',
+    },
+    proposal: {
+      give: {
+        // @ts-expect-error BoardRemote is not assignable to Brand<any>
+        ATOM: { brand: ATOM, value: 100n },
+      },
+      exit: { waived: null },
+    },
+  });
+  t.like(wd.getLatestUpdateRecord(), {
+    status: { id: 'request-deposit-success', numWantsSatisfied: 1 },
+  });
+
+  await t.throwsAsync(
+    wd.executeOffer({
+      id: 'request-deposit-failure-no-want-allowed',
+      invitationSpec: {
+        source: 'continuing',
+        previousOffer: 'request-account',
+        invitationMakerName: 'Deposit',
+      },
+      proposal: {
+        give: {
+          // @ts-expect-error BoardRemote is not assignable to Brand<any>
+          ATOM: { brand: ATOM, value: 100n },
+        },
+        want: {
+          // @ts-expect-error BoardRemote is not assignable to Brand<any>
+          BLD: { brand: BLD, value: 100n },
+        },
+        exit: { waived: null },
+      },
+    }),
+    {
+      message: /proposal: want(.*?)Must be: {}/,
+    },
+  );
+
+  await t.throwsAsync(
+    wd.executeOffer({
+      id: 'request-deposit-failure-two-give-amounts',
+      invitationSpec: {
+        source: 'continuing',
+        previousOffer: 'request-account',
+        invitationMakerName: 'Deposit',
+      },
+      proposal: {
+        give: {
+          // @ts-expect-error BoardRemote is not assignable to Brand<any>
+          ATOM: { brand: ATOM, value: 100n },
+          // @ts-expect-error BoardRemote is not assignable to Brand<any>
+          BLD: { brand: BLD, value: 100n },
+        },
+        exit: { waived: null },
+      },
+    }),
+    {
+      message: /proposal: give: Must not have more than 1 properties/,
+    },
+  );
+
+  await t.throwsAsync(
+    wd.executeOffer({
+      id: 'request-deposit-failure-unknown-issuer',
+      invitationSpec: {
+        source: 'continuing',
+        previousOffer: 'request-account',
+        invitationMakerName: 'Deposit',
+      },
+      proposal: {
+        give: {
+          // @ts-expect-error BoardRemote is not assignable to Brand<any>
+          DAI_axl: { brand: DAI_axl, value: 100n },
+        },
+        exit: { waived: null },
+      },
+    }),
+    {
+      message: /brand(.*?)not registered/,
+    },
+  );
+
+  await t.throwsAsync(
+    wd.executeOffer({
+      id: 'request-deposit-failure-transfer-packet-timeout',
+      invitationSpec: {
+        source: 'continuing',
+        previousOffer: 'request-account',
+        invitationMakerName: 'Deposit',
+      },
+      proposal: {
+        give: {
+          // @ts-expect-error BoardRemote is not assignable to Brand<any>
+          ATOM: { brand: ATOM, value: 504n },
+        },
+        exit: { waived: null },
+      },
+    }),
+    {
+      message: 'Deposit failed, payment returned.',
```

Comment:
again, it would be good to have evidence that the right vbank messages went over the bridge

ah... but we do have `payouts` below